src/doc/rustc/src/platform-support/nto-qnx.md

```diff
@@ -134,14 +139,18 @@ export build_env='
     CFLAGS_aarch64-unknown-nto-qnx710=-Vgcc_ntoaarch64le_cxx
     CXX_aarch64-unknown-nto-qnx710=qcc
     AR_aarch64_unknown_nto_qnx710=ntoaarch64-ar
+    CC_aarch64-unknown-nto-qnx710_iosock=qcc
+    CFLAGS_aarch64-unknown-nto-qnx710_iosock=-Vgcc_ntoaarch64le_cxx
+    CXX_aarch64-unknown-nto-qnx710_iosock=qcc
```

Comment:
I think the `CXX` variable should use `_` not `-`. Or at least [`src/ci/docker/host-x86_64/dist-aarch64-linux/Dockerfile:24`](https://github.com/rust-lang/rust/blob/5bbbc0938cfeb3dbd348d5258ddb6025bc8398c7/src/ci/docker/host-x86_64/dist-aarch64-linux/Dockerfile#L24) says `CXX_aarch64_unknown_linux_gnu=aarch64-unknown-linux-gnu-g++`.

Probably the same for the other variables here too.